packages/utils/interactive-hooks/src/useAsPressable.types.ts

```diff
@@ -29,3 +30,15 @@ export type IPressableHooks<T extends object> = {
   props: IWithPressableEvents<T>;
   state: IPressableState;
 };
+
+/**
+ * The hover props on RN 0.63 don't exist in react-native, only on the desktop SKUs, but we need them for hooking
+ * mouse events on windows and macos. This adds them to the props type, stripping them from the base pressable type as
+ * future proofing for when we upgrade to 0.64+.
+ *
+ * Similarly the focus methods exist at the View level but are not exposed on pressable props in 0.63. As a result they
+ * need a similar treatment
```

Comment:
Is this true for windows as well? I know it's not for macOS but I thought it was for windows?